src/site/_filters/remove-drafts.js

```diff
@@ -13,7 +13,7 @@ module.exports = function removeDrafts(topics, lang) {
     // Remove draft posts from a topic.
     const posts = topic.pathItems.filter((slug) => {
       const post = findByUrl(`/${lang}/${slug}/`);
-      return post && !post.data.draft;
+      return (post && !post.data.draft) || (slug.title && slug.url);
```

Comment:
@robdodson updated this filter to accept the "slugs" that are actually objects pointing to external links. Two problems:

* "slugs" is no longer conceptually accurate (and that word appears to be used heavily throughout the codebase)
* This filter now has a purpose that is broader than removing drafts. I think that might be easily solved by renaming this to something along the lines of "remove invalid posts"